lesson04/assignment/index.html

```diff
@@ -4,12 +4,60 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="https://kit.fontawesome.com/e418cb4dcb.js"></script>
   <link rel="stylesheet" href="css/style.css">
-  <title>Document</title>
+  <title>Wrix</title>
 </head>
 <body>
-  <main class="main">
+  <!--Header, to include background image-->
+  <header>
+  </header>
+  <!--Nav menu-->
+  <nav class="nav">
+    Home /
```

Comment:
This should be a link to the current page, right? Or do you want it not a real link, to indicate to user that it's their current location?